src/core/public/plugins/plugins_service.test.ts

```diff
@@ -34,7 +34,7 @@ import { applicationServiceMock } from '../application/application_service.mock'
 import { overlayServiceMock } from '../overlays/overlay_service.mock';
 import { chromeServiceMock } from '../chrome/chrome_service.mock';
 import { fatalErrorsServiceMock } from '@kbn/core-fatal-errors-browser-mocks';
-import { uiSettingsServiceMock } from '../ui_settings/ui_settings_service.mock';
+import { uiSettingsServiceMock } from '@kbn/core-ui-settings-browser-mocks';
```

Comment:
nit: imports by module ID should be placed before relative-path imports (I don't know why ESLint does not apply that rule in the `.test.ts` files 🤷

Reply:
> I don't know why ESLint does not apply that rule in the .test.ts files 

That's because in some situations (e.g importing `.mocks.ts` files performing `jest.doMock`), this rule can't be properly applied without breaking the mocking

Reply:
Ah, thanks! The comment was carried over from core.